### DISCUSSION TITLE
KP.3.1.1(S:S31del)+S:S31F

### DISCUSSION THREAD

**Discussion by Memorablea** [Memorablea/COVID-19-#103] [Q&A] [answered]
KP.3.1.1+S:S31F
ping @FedeGueli 
![image](https://github.com/Memorablea/COVID-19-/assets/125779492/62360d02-dc91-499b-9903-92279565db64)
![image](https://github.com/Memorablea/COVID-19-/assets/125779492/c92fefa1-a679-40bc-a284-0242764127cf)

_Originally posted by @Memorablea in https://github.com/Memorablea/COVID-19-/issues/102#issuecomment-2182460025_

**Comment by FedeGueli** [comment, marked as answer]
Artefact. Denmark has issues with deletions.
